lms/djangoapps/student_profile/views.py

```diff
@@ -176,7 +176,7 @@ def learner_performance(request, course_id=None, student_id=None):
         "api_endpoint": "profile_statistics",
         "key": getattr(settings, 'EDEOS_API_KEY'),
         "secret": getattr(settings, 'EDEOS_API_SECRET'),
-        "base_url": 'http://195.160.222.156/api/point/v1/'
+        "base_url": getattr(settings, 'EDEOS_API_STATISTICS')
```

Comment:
please name the property EDEOS_API_STATISTICS_URL

Reply:
fixed